website/docs/api/plugins/plugin-content-blog.md

```diff
@@ -94,27 +100,27 @@ type ReadingTimeFunctionOption = (params: {
 type FeedType = 'rss' | 'atom' | 'json';
 ```
 
-## Example configuration {#ex-config}
+### Example configuration {#ex-config}
 
-Here's an example configuration object.
-
-You can provide it as [preset options](#ex-config-preset) or [plugin options](#ex-config-plugin).
+You can configure this plugin through preset options or plugin options.
 
 :::tip
 
-Most Docusaurus users configure this plugin through the [preset options](#ex-config-preset).
+Most Docusaurus users configure this plugin through the preset options.
 
 :::
 
-```js
+```js config-tabs
+// preset option name: blog
+// plugin name: @docusaurus/plugin-content-blog
```

Comment:
If you are reading the source code and think this detracts from readability, don't get confused by this: read on! This will be processed by a remark plugin and turned into a full preset/plugin config.